vnext/Desktop.IntegrationTests/RNTesterIntegrationTests.cpp

```diff
@@ -16,6 +17,10 @@ std::wstring ToString<TestStatus>(const TestStatus &status) {
 
 } // namespace Microsoft::VisualStudio::CppUnitTestFramework
 
+TEST_MODULE_INITIALIZE(InitModule) {
+  Microsoft::React::SetFeatureGate("Office.CXE.React.UseWinRTWebSocket");
```

Comment:
Nit: Could we avoid Office naming for the FG for this?

Reply:
We actually can. I thought we could "pass" the Office FG directly to this API, but that's not how these would be used.
The names don't really have to match, but  I'll keep the suffixes identical fore ease of tracking.